src/signals/incident/components/form/MapSelectors/Asset/AssetList/AssetList.tsx

```diff
@@ -52,45 +51,30 @@ const AssetList: FunctionComponent<AssetListProps> = ({
   className,
   featureTypes,
 }) => {
-  const item = useMemo(() => {
-    const { id, type, isReported } = selection
-    const { description, icon }: Partial<FeatureType> =
-      featureTypes.find(({ typeValue }) => typeValue === type) ?? {}
+  const { id, type, isReported } = selection
+  const { description, icon }: Partial<FeatureType> =
+    featureTypes?.find(({ typeValue }) => typeValue === type) ?? {}
 
-    const label = [description, id].filter(Boolean).join(' - ')
+  const label = [description, id].filter(Boolean).join(' - ')
 
-    const baseItem = {
-      id,
-      label,
-    }
-
-    return {
-      ...baseItem,
-      iconUrl: icon ? icon.iconUrl : '',
-      isReported,
-    }
-  }, [featureTypes, selection])
+  if (!id) return null
 
   return (
     <IconList data-testid="assetList" className={className}>
       <IconListItem
-        key={item.id}
-        id={
-          item.isReported
-            ? `assetListItem-${item.id}-reported`
-            : `assetListItem-${item.id}`
-        }
-        iconUrl={item.iconUrl}
-        isReported={item.isReported}
+        key={id}
+        id={isReported ? `assetListItem-${id}-reported` : `assetListItem-${id}`}
+        iconUrl={icon?.iconUrl}
+        isReported={isReported}
       >
         <ItemWrapper>
           <StyledLabel>
-            <div>{item.label}</div>
-            {item.isReported && <StyledDiv>Is gemeld</StyledDiv>}
```

Comment:
This div is necessary for the flex layout.

Reply:
Don't think so; `StyledLabel` is not a flex element.

Reply:
Perhaps you are right, but the styling is definitely messed up if you remove the div